src/osd/PG.h

```diff
@@ -381,8 +385,7 @@ class PG : public DoutPrefixProvider, public PeeringState::PeeringListener {
   void scrub(epoch_t queued, ThreadPool::TPHandle& handle)
   {
     // a new scrub
-    scrub_queued = false;
-    forward_scrub_event(&ScrubPgIF::initiate_regular_scrub, queued, "StartScrub"sv);
+    forward_scrub_event(&ScrubPgIF::initiate_regular_scrub, queued, "StartScrub");
```

Comment:
Why are we changing `forward_scrub_event()` in some places, not all?

Reply:
Just a coincidence - the sv attribute was removed by someone along the way (and I didn't object too much). Thus , it wasn't there in the copied code.